release/_publishing.sh

```diff
@@ -1,5 +1,49 @@
 #!/bin/bash
 
+function add_fixed_issues_to_project_version {
+	local fixed_issues=$(< "${_BUILD_DIR}/release/release-notes.txt")
+
+	IFS=',' read -r -a fixed_issues_array <<< "${fixed_issues}"
+
+	local fixed_issues_array_length="${#fixed_issues_array[@]}"
+
+	local quarter_length=$((fixed_issues_array_length / 4))
```

Comment:
@kiwm, do we really need 2 pairs of `(`? Just use a single one isn't enough, like in `$(...)`?

Additionally, can we rename `quarter_length` to `fixed_issues_array_part_length`?